dart-mode.el

```diff
@@ -42,11 +42,28 @@
 ;;; Code:
 
 (require 'cc-mode)
-(require 'cc-langs)
-(eval-when-compile (require 'cl))
+(eval-when-compile
+  (require 'cc-langs)
+  (require 'cc-fonts))
 
 (eval-and-compile (c-add-language 'dart-mode 'java-mode))
 
+(require 'dash)
+(require 'flycheck)
+(require 'json)
+(require 's)
```

Comment:
The only function you're using from this is `s-present?`, which is very simple. This doesn't warrant a new dependency.